### DISCUSSION TITLE
[Question] When trying to install Europe, or any place, after a fresh install, the process stops here: Check Image - Please Help

### DISCUSSION THREAD

**Discussion by Semern55555** [kubilus1/autoortho#373] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: [Windows 11]
- XPlane Version: [XP11 11.55]
- AutoOrtho Version [0.6.0]
- Filesystem type (NTFS)

### question

**Describe your question**
When trying to install Europe, or any place, after a fresh install, the process stops here: Check Image - Please Help
(Both Python and WinFSP is installed, Dokan aswell, but have changed CFG so that it uses WinFSP by default)

**To Reproduce**
Steps to reproduce the behavior, if applicable:
1. Go to 'AutoOrtho'
2. Click on 'Install Europe'
4. See error at 'cmd'

**Screenshots**

![image](https://github.com/kubilus1/autoortho/assets/81387651/667af68b-041d-44a7-b70d-700383c6cd2a)

![image](https://github.com/kubilus1/autoortho/assets/81387651/7ec84374-12cb-4b82-a0a8-122b4aa916fc)

**Comment by Jeff2122** [comment]
I see it working, and it is probably in the middle of unpacking zips by the looks of you pictures above. How long did you wait? I can't see your Setup tab, did you set the folders correctly?

**Reply by Semern55555** [reply]
Tried multiple times, at one time I waited around 20 minutes. Here is a picture of my Setup tab:

![image](https://github.com/kubilus1/autoortho/assets/81387651/e549942e-a743-4990-98b5-11aebab6e088)

**Comment by Jeff2122** [comment]
I don't see anything wrong with that. I wouldn't put my cache on my system drive, but that's just me. 
Did you run the installer as administrator? It might help. 
It sounds like it might have issues unpacking the zips for some reason. I haven't done it in a week so it's not fresh in my mind what it looks like. 
I wonder if there is something about Windows security settings on your folder.
I know you are on the edge of frustration, but don't give up. 
Let me find out which download is smallest and I'll do it to see. I only have NA and EUR installed at this time.

**Reply by Semern55555** [reply]
Hehe, of course I wont give up just yet! Ill try to move my cache to the same drive as X-Plane then. Yes, always running it as administrator. Might be Windows security then as you say, but I have no clue on that. Ill try with cache on another drive, and ill wait for you too see aswell! :) Thanks for trying to help me!

**Comment by Jeff2122** [comment]
![image](https://github.com/kubilus1/autoortho/assets/124176597/f4d3b6d7-5460-4b23-9024-cc92359bd2f0)

I made my Temp directory same as yours and downloaded Australia AO pack. Watched it download 13 zip files plus one overlay zip, unpack and set up the folders. But I don't have Australia scenery in XP12 yet so can't test the ortho there. I'm sure it will work though. 
I will keep thinking. You don't know how to check credentials on your folders, do you? Right click, properties, Security tab, should see Administrators have Full Control.

**Reply by Semern55555** [reply]
Yeah, tried again, same problem, checked Security tab, full control enabld. Really strange, just saw on the org forums that someone had same problem, but got fixed at a later time, probably server issues, but it is working for you, so it is strange! Maybe ill just try again tomorow? Anyway, appreciate you trying to help me!

**Comment by Jeff2122** [comment]
[InstallAUS_PAC.log.txt](https://github.com/kubilus1/autoortho/files/12504751/InstallAUS_PAC.log.txt)

**Comment by Jeff2122** [comment]
The above file is an excerpt from my log from that exercise above. It doesn't show the download speed, but it doesn't matter. Yours went all the way until it started assembling the file structure then halted. That's the line "INFO:downloader:Begin parsing info_dict for Australia Pacific Ortho Set".

**Reply by Semern55555** [reply]
Yeah, it is really strange, huh, haha

**Comment by Jeff2122** [comment]
Do you know Discord? There is a guy who is helping folks and I'm going to start checking in there. I am not a big social networker so I might not like it and continue. Not sure if it is by invite only but will find out.

**Reply by Semern55555** [reply]
I do have Discord! Let me know if you need my username!

**Comment by Jeff2122** [comment]
@Semern55555  How are you getting along? Did you get some help?

**Reply by Semern55555** [reply]
I am getting help, currently trying some different things to try and fix it, ill let you know!

**Reply by Semern55555** [reply]
Sorry, totaly forgot to tell, but time was the problem, just let it be overnight, and it works, love it! Thanks again for the help! See you in the skies!

**Comment by DaveGaming01** [comment]
Has a fix for this been found yet? I'm having the same issue right now

**Reply by Semern55555** [reply]
Hi, yes, there has been found a fix, the problem was just time, let it be, can take multiple hours, I just let it install over night, and it worked! Hope it helps!

**Reply by DaveGaming01** [reply]
Well hopefully you're right

Thank you for the quick response and have a good night 😃

**Reply by Semern55555** [reply]
Likewise! ☺️
